Jenkinsfile

```diff
@@ -59,20 +62,58 @@ pipeline {
                                 source ./.cicd_bootstrap.sh
                                 source "${CICD_ROOT}/deploy_ephemeral_env.sh"
                                 source "${CICD_ROOT}/cji_smoke_test.sh"
+
+                                > variables
+                                echo "NAMESPACE:$NAMESPACE" >> variables
+                                echo "IMAGE_TAG:$IMAGE_TAG" >> variables
                             '''
                         }
-                    }
-                    post {
-                        failure {
-                            slackSend  channel: '@eshamard', color: "danger", message: "Smoke tests failed in Compliance PR check. <${env.ghprbPullLink}|PR link>  (<${env.BUILD_URL}|Build>)"
-                        }
-                        unstable {
-                            slackSend  channel: '@eshamard', color: "warning", message: "Smoke tests failed in Compliance PR check. <${env.ghprbPullLink}|PR link>  (<${env.BUILD_URL}|Build>)"
+
+                        script {
+                            FILE_CONTENTS = readFile('variables')
+                            flags_map = [:]
+                            flags = FILE_CONTENTS.split("\n")
+                            for (i in flags) {
+                                s=i.split(':')
+                                if (s.length == 2) {
+                                    flags_map[s[0]] = "${s[1]}"
+                                } else {
+                                    flags_map[s[0]] = ""
+                                }
+                            }
+
+                            env.NAMESPACE = flags_map['NAMESPACE']
+                            env.IMAGE_TAG = flags_map['IMAGE_TAG']
```

Comment:
I would avoid doing all this if the purpose is "only" to get the namespace - and the image_tag. They can both be saved as groovy variables.